clients/src/main/java/org/apache/kafka/clients/admin/AbstractOptions.java

```diff
@@ -23,24 +23,24 @@
  */
 public abstract class AbstractOptions<T extends AbstractOptions> {
 
-    protected Integer timeoutMs = null;
+    protected Integer apiTimeoutMs = null;
 
     /**
-     * Set the request timeout in milliseconds for this operation or {@code null} if the default request timeout for the
+     * Set the api timeout in milliseconds for this operation or {@code null} if the default api timeout for the
      * AdminClient should be used.
      */
     @SuppressWarnings("unchecked")
-    public T timeoutMs(Integer timeoutMs) {
-        this.timeoutMs = timeoutMs;
+    public T apiTimeoutMs(Integer apiTimeoutMs) {
+        this.apiTimeoutMs = apiTimeoutMs;
         return (T) this;
     }
 
     /**
-     * The request timeout in milliseconds for this operation or {@code null} if the default request timeout for the
+     * The api timeout in milliseconds for this operation or {@code null} if the default api timeout for the
      * AdminClient should be used.
      */
-    public Integer timeoutMs() {
-        return timeoutMs;
+    public Integer apiTimeoutMs() {
```

Comment:
These changes are incompatible; we need to keep the existing API. I think it's fine to update the javadoc.